src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -1,218 +1,248 @@
-	void testStringToIntValidLowercaseRomanSymbol(String romanNumeral, int expected) {
-		int actual = convertFromStringToInt(romanNumeral);
-		assertEquals(expected, actual);
-	}
-	
-	@ParameterizedTest
-	@ValueSource(strings = {"@", "V!", "123", "ten", "vi"})
-	void testStringToIntInvalidRomanNumeral(String invalidInput) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(invalidInput));
-	}
-	
-	@ParameterizedTest
-	@ValueSource(strings = {"IIII", "VIIII", "xxxx", "CMXCIX", "IM"})
-	void testStringToIntNonStandardRomanNumeral(String invalidInput) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(invalidInput));
-	}
-
-	@ValueSource(ints = {0, -1, 10000})
-	void testConvertFromIntToStringThrowsExceptionForOutOfRange(int n) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromIntToString(n));
-	}
 	
-	@ParameterizedTest
-	@ValueSource(strings = {"DCCCC", "IIII", "VIIII", "XXXX"})
-	void testThrowsExceptionForNonStandardForm(String s) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromStringToInt(s));
-	}
 	
+	//Does the test convert Strings to ints?
 	@ParameterizedTest
-	@ValueSource(ints = {-1, 0, 10000})
-	void testOutOfBoundsNumbers(int num) {
-		assertThrows(IllegalArgumentException.class,
-				() -> convertFromIntToString(num));
+	@CsvSource({"II,2.0","III,4", "VI,5", "XII,12"})
+	void string2IntConvert(String expected, int actual)
+	{
+		int input = RomanNumeral.convertFromStringToInt(expected);
+		
+		
+		assertEquals(input,actual);
 	}
 	
-	@ParameterizedTest
-	@CsvSource({"II,2", "III,3", "XI,11", "lii,52", "DCXIII,613"})
-	void testStringToIntAdditivePrinciple(String additiveNumeral, int expected) {
-		int actual = convertFromStringToInt(additiveNumeral);
-		assertEquals(expected, actual);
-	}
 	
-	@ParameterizedTest
-	@CsvSource({"iv, 4", "IV,4", "IX,9", "XL,40", "XC,90", "CD,400", "CM,900"})
-	void testStringToIntSubtractivePrinciple(String subtractiveNumeral, int expected) {
-		int actual = convertFromStringToInt(subtractiveNumeral);
-		assertEquals(expected, actual);
-	}
+	//Does the test convert ints to Strings?
+	@ParameterizedTest
+	@CsvSource({"4,four","10,V", "18, XVIII", "7,VII" })
+	void int2StringConvert(int expected, String actual)
+	{
+		String input = RomanNumeral.convertFromIntToString(expected);
+		//assertThrows(IllegalArgumentException.class);
+		assertEquals(input,actual);
+	}	
 	
-	// Testing for convertFromIntToString
 	@ParameterizedTest
-	@CsvSource({"1,I", "5,V", "10,X", "50,L", "100,C", "500,D", "1000,M"})
-	void testIntToStringIsValidRomanSymbol(int n, String expected) {
-		String actual = convertFromIntToString(n);
-		assertEquals(expected, actual);
+	@ValueSource(ints = {-4,-10,0,-18}, strings = {"four","V","XVIII"})
+	void testExceptionsThrown(int i, String s) {
+		assertThrows(IllegalArgumentException.class,() -> int2StringConvert(i,s));
```

Comment:
I don't understand why that calls another method (that is called only by this method). Copy that code into here and eliminate the other method.